modules/about-redhat-openshift-gitops.adoc

```diff
@@ -0,0 +1,26 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/understanding-openshift-gitops.adoc
+
+[id="about-redhat-openshift-gitops_{context}"]
+= About {gitops-title}
+
+{gitops-title} ensures consistency in applications when you deploy them to different clusters in different environments, like: development, stage, and production. {gitops-title} organizes the deployment process around the code repositories and makes them the central element. It always has at least two repositories:
+
+  . Application repository with the source code
+  . Environment configuration repository that defines the desired state of the application
+
+These repositories contain a declarative description of the infrastructure you need in your specified environment. They also contain an automated process to make your environment match the described state.
+
+{gitops-title} uses Argo CD to maintain cluster resources. Argo CD is an open-source declarative tool for the continuous integration and continuous deployment (CI/CD) of applications. Red Hat OpenShift GitOps implements Argo CD as a controller so that it continuously monitors application definitions and configurations defined in a Git repository. Then, Argo CD compares the specified state of these configurations with their live state on the cluster. Argo CD reports any configurations that deviate from their specified state. These reports allow administrators to automatically or manually resync configurations to the defined state. Therefore, Argo CD enables you to deliver global custom resources, like the resources that are used to configure OpenShift Container Platform clusters.
+
+[id="key-features_{context}"]
+== Key features
+
+{gitops-title} helps you:
+
+* Ensure that the clusters have similar states for configuration, monitoring, and storage
+* Recover or recreate clusters from a known state
+* Apply or revert configuration changes to multiple OpenShift Container Platform clusters
```

Comment:
OpenShift Container Platform/{product-title}

Reply:
See comment above. May I keep it the way it is, fully writing out `OpenShift Container Platform`?